release.py

```diff
@@ -415,15 +415,18 @@ def cf_hash(opts, pargs):
 if __name__ == "__main__":
     parser = create_option_parser()
     (opts, pargs) = parser.parse_args()
-    
+
     if len(pargs) < 2:
         parser.error("Improper usage. Too few arguments!")
     if len(pargs) > 2:
         parser.error("Improper usage. Too many arguments!")
-    
+
+    if opts.release and opts.pre_release:
+        parser.error("Please re-run the command specifying either release or pre_release.")
```

Comment:
"reason for error" -> user entered both release and pre_release"
" what to do to fix it" -> re-run